cassiopeia/preprocess/doublet_utils.pyx

```diff
@@ -65,18 +63,14 @@ def filter_intra_doublets(
         "cellBC"
     ].unique()
 
-    if verbose:
-        logging.info(
-            f"Filtered {len(doublet_list)} Intra-Lineage Group Doublets of "
-            + str(len(molecule_table["cellBC"].unique()))
-        )
+    logger.debug(
+        f"Filtered {len(doublet_list)} Intra-Lineage Group Doublets of "
+        + str(len(molecule_table["cellBC"].unique()))
+    )
```

Comment:
What's the motivation for using `DEBUG` rather than `INFO` for these types of summaries in the log file?

Also, since we're updating logging functionality, can we log the parameter passed to this function as well?

Reply:
I just changed all the logging performed in the following manner
```python
if verbose:
    logging.info('...')
```
to
```python
logger.debug('...')
```
(so i set all the logging statements that used to print only when `verbose=True` into debug messages)
We can change it if we want. No particular technical reason for this.

Reply:
Also, I believe the arguments to that function is already logged before the function call.
```python
    if inter_doublet_threshold:
        logger.info(
            f"Filtering out inter-lineage group doublets with proportion {inter_doublet_threshold}..."
        )
        allele_table = d_utils.filter_inter_doublets(
            allele_table, rule=inter_doublet_threshold
        )

```